### PR TITLE
Adding Initial Implementation.

### PR DESCRIPTION
**Pull request by DivyaJyotiDas**

### CODE REVIEW

File: assignment.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        if i < len(nodes_list) and nodes_list[i] is not None:
+            current.right = Forest(nodes_list[i])
+            queue.append(current.right)
+        i += 1
+    return forest
+
+
+if __name__ == '__main__':
+    # TestCase #01
+    nodes_list = [5, 4, 8, 11, None, 9, 4, None, None, 7, 2, None, None, None, 5, 1]
```
Comment (manidhayal) [inline assignment.py:87]: This is valid input from the problem statement given. Please use this to test
[5, 4, 8, 11, None, 9, 4, -7, 2, None, None, 5, 1]

Comment (DivyaJyotiDas) [reply assignment.py:87]: Do you think with this list, you can ever get the path that is mentioned in the problem statement ? 
```
[5, 4, 8, 11, None, 9, 4, -7, 2, None, None, 5, 1]
”5,4,11,2;5,8,9;5,8,4,5”
```

Comment (manidhayal) [reply assignment.py:87]: Yes I have working program

Comment (DivyaJyotiDas) [reply assignment.py:87]: good, raise a PR will review.

**Review by manidhayal — APPROVED** [review]

File: assignment.py (hunk 1 above)
Comment (manidhayal) [reply assignment.py:87]: I have added to main mine and your code as well.. you can check the main branch itself..

**Pull request merged**